docs/src/examples/components/StatusIndicator/Types/StatusIndicatorExample.shorthand.tsx

```diff
@@ -0,0 +1,34 @@
+import * as React from 'react'
+import { StatusIndicator } from '@stardust-ui/react'
+
+const StatusIndicatorShorthandExample = () => (
+  <div style={{ display: 'flex', justifyContent: 'space-around' }}>
+    <StatusIndicator />
+    <StatusIndicator
+      icon={{ name: 'check', variables: { color: 'white', backgroundColor: 'green' } }}
+      title="Available"
+    />
+    <StatusIndicator
+      icon={{ name: 'clock', variables: { color: 'white', backgroundColor: 'yellow' } }}
+      title="Away"
+    />
+    <StatusIndicator
+      icon={{ name: 'clock', variables: { color: 'white', backgroundColor: 'yellow' } }}
+      title="Be right back"
+    />
+    <StatusIndicator
+      icon={{ name: '', variables: { color: 'white', backgroundColor: 'red' } }}
+      title="Busy"
+    />
+    <StatusIndicator
+      icon={{ name: 'minus', variables: { color: 'white', backgroundColor: 'red' } }}
+      title="Do not disturb"
+    />
+    <StatusIndicator
+      icon={{ name: '', variables: { color: 'white', backgroundColor: 'grey' } }}
```

Comment:
Feels odd to pass an empty name string here.  Let's investigate ways to improve the API.

Reply:
Removed the empty icon for now. We can revisit this later.

---
In reply to: [215418325](https://github.com/stardust-ui/react/pull/168#discussion_r215418325) [](ancestors = 215418325)